cnf/m4/ax_compiler_flags_cflags.m4

```diff
@@ -69,12 +77,9 @@ AC_DEFUN([AX_COMPILER_FLAGS_CFLAGS],[
             -Wall dnl
             -Wextra dnl
             -Wundef dnl
-            -Wnested-externs dnl
```

Comment:
First thought this got remove, but apparently it was just moved to a C specific section (though we always use a C compiler anyway, don't we?!?)